test/revert.spec.ts

```diff
@@ -5800,7 +5810,7 @@ describe(`Reverts (Seaport v${VERSION})`, function () {
     let seller: Wallet;
     let buyer: Wallet;
 
-    before(async () => {
+    beforeEach(async () => {
```

Comment:
This is necessary because `loadFixture` uses snapshots to revert the network to a given state. This combines with the fact that mocha's `before` hooks are executed before any `beforeEach` (even the ones that are at an upper level). So here what happens is:

- This `before` executes first
- The `beforeEach` reverts to the fixture's snapshots and undoes anything done here

Using a `beforeEach` fixes that. It's a bit more wasteful, but I checked and it's still worth it, performance-wise.